publication-commons/src/main/java/no/unit/nva/publication/model/utils/CuratingInstitutionsUtil.java

```diff
@@ -42,7 +47,9 @@ private static Stream<URI> getOrganizationIds(Contributor contributor) {
     }
 
     private static Stream<Contributor> getVerifiedContributors(EntityDescription entityDescription) {
-        return entityDescription.getContributors()
+        return Optional.ofNullable(entityDescription)
+                   .map(EntityDescription::getContributors)
+                   .orElse(Collections.emptyList())
```

Comment:
We don't do this automatically in data-model?

Reply:
not sure. tried to search a bit. we do this several places for sure

Reply:
Should probably look into the model issues at a deeper level

Reply:
will take this in this pr and then I can do fixes in next PR if needed